### PR TITLE
hammer_cli{_foreman} 3.0

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

**Comment by Odilhao** [comment]
```
[2021-08-09T13:49:26.920Z] fatal: [rubygem-hammer_cli]: FAILED! => changed=false 

[2021-08-09T13:49:26.920Z]   changelog_epoch_version_release: 3.0.0-1

[2021-08-09T13:49:26.920Z]   msg: changelog entry missing for 3.0.0-0.1.pre.develop

[2021-08-09T13:49:26.920Z]   specfile_epoch_version_release: 3.0.0-0.1.pre.develop
```
We probably want to resolve this version condition

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 1]
```diff
@@ -5,8 +5,8 @@
 %global gem_name hammer_cli
 
 %global release 1
-%global prereleasesource pre.develop
-%global prerelease %{?prereleasesource:.}%{?prereleasesource}
+#%global prereleasesource pre.develop
+#%global prerelease %{?prereleasesource:.}%{?prereleasesource}
```
Comment (ehelms) [inline packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:9]: IIRC -- you should just delete this or you have to do `%%` to disable the macro, comments dont actually disable it

Comment (pcreech) [reply packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec]: Yay rpm....  fixeded

**Review by ehelms — APPROVED** [review]

**Comment by pcreech** [comment]
Repoclosure failure expected, due to repos not being published yet.

**Pull request merged**